tools/tf2openapi/README.md

```diff
@@ -0,0 +1,2 @@
+The main purpose of this tool is to enable automatically generating OpenAPI 3.0 specifications for prediction HTTP requests from TensorFlow SavedModel files containing signature definitions (SignatureDefs) for models. One use case for the OpenAPI specification involves generating sample payloads in a UI.
```

Comment:
Would be nice to have a little bit of Markdown in the readme.

Something like

* Overview
* Example
* Caveats